app/models/etsource/inputs.rb

```diff
@@ -1,20 +1,23 @@
 module Etsource
-
   class Inputs
-    def initialize(etsource = Etsource::Base.instance)
-      @etsource = etsource
+    INPUTS = {
```

Comment:
We can also do this:

```ruby
Atlas::InitializerInput.to_s.gsub(/.*\:\:/, '').constantize # => InitializerInput
```

But that seems like a really horrible unstable solution 😆